### ISSUE TITLE
Can't create/find/open settings file on Windows

### ISSUE THREAD

**Issue by YousefSaber** [gtasa-savegame-editor/gtasa-savegame-editor#71]
```
PS C:\Users\youse\Desktop\deploy> ls


    Directory: C:\Users\youse\Desktop\deploy


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        19/12/2020  09:20 PM       12054024 savegame-editor-3.3-SNAPSHOT-jar-with-dependencies.jar


PS C:\Users\youse\Desktop\deploy> java -version
java version "15.0.2" 2021-01-19
Java(TM) SE Runtime Environment (build 15.0.2+7-27)
Java HotSpot(TM) 64-Bit Server VM (build 15.0.2+7-27, mixed mode, sharing)
PS C:\Users\youse\Desktop\deploy> java -jar .\savegame-editor-3.3-SNAPSHOT-jar-with-dependencies.jar
java.lang.Error: Invalid memory access
        at com.sun.jna.Native.invokeInt(Native Method)
        at com.sun.jna.Function.invoke(Function.java:426)
        at com.sun.jna.Function.invoke(Function.java:361)
        at com.sun.jna.Library$Handler.invoke(Library.java:265)
        at com.sun.proxy.$Proxy0.SHGetSpecialFolderPath(Unknown Source)
        at nl.paulinternet.gtasaveedit.Settings.getSettingsFile(Settings.java:69)
        at nl.paulinternet.gtasaveedit.Settings.<clinit>(Settings.java:23)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.<init>(MainWindow.java:30)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.getInstance(MainWindow.java:20)
        at nl.paulinternet.gtasaveedit.view.Main.main(Main.java:55)
Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class nl.paulinternet.gtasaveedit.Settings
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.<init>(MainWindow.java:30)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.getInstance(MainWindow.java:20)
        at nl.paulinternet.gtasaveedit.view.window.ExceptionDialog.<init>(ExceptionDialog.java:47)
        at nl.paulinternet.gtasaveedit.view.Main.main(Main.java:68)
PS C:\Users\youse\Desktop\deploy>
```

**Comment by lfuelling** [comment]
Hi, you need to use Java 11.

**Comment by YousefSaber** [comment]
@lfuelling installed Java 11 from [AdoptOpenJDK](https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=hotspot) because the one from the official [java](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) requires some license fees and yeah the problem still persists. 
```
PS C:\Users\youse\Desktop\deploy> java -version
openjdk version "11.0.10" 2021-01-19
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.10+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.10+9, mixed mode)
PS C:\Users\youse\Desktop\deploy> ls


    Directory: C:\Users\youse\Desktop\deploy


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        19/12/2020  09:20 PM       12054024 savegame-editor-3.3-SNAPSHOT-jar-with-dependencies.jar


PS C:\Users\youse\Desktop\deploy> java -jar .\savegame-editor-3.3-SNAPSHOT-jar-with-dependencies.jar
java.lang.Error: Invalid memory access
        at com.sun.jna.Native.invokeInt(Native Method)
        at com.sun.jna.Function.invoke(Function.java:426)
        at com.sun.jna.Function.invoke(Function.java:361)
        at com.sun.jna.Library$Handler.invoke(Library.java:265)
        at com.sun.proxy.$Proxy0.SHGetSpecialFolderPath(Unknown Source)
        at nl.paulinternet.gtasaveedit.Settings.getSettingsFile(Settings.java:69)
        at nl.paulinternet.gtasaveedit.Settings.<clinit>(Settings.java:23)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.<init>(MainWindow.java:30)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.getInstance(MainWindow.java:20)
        at nl.paulinternet.gtasaveedit.view.Main.main(Main.java:55)
Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class nl.paulinternet.gtasaveedit.Settings
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.<init>(MainWindow.java:30)
        at nl.paulinternet.gtasaveedit.view.window.MainWindow.getInstance(MainWindow.java:20)
        at nl.paulinternet.gtasaveedit.view.window.ExceptionDialog.<init>(ExceptionDialog.java:47)
        at nl.paulinternet.gtasaveedit.view.Main.main(Main.java:68)
PS C:\Users\youse\Desktop\deploy>
```

**Comment by lfuelling** [comment]
I'm not sure where that `java.lang.Error: Invalid memory access` is coming from but I have never tested the settings file handling on Windows (because I don't use it).

I'll try to publish a fix later, would you be willing to help debugging this?

**Comment by YousefSaber** [comment]
@lfuelling I'm not experienced with programming but of course I'll try to help you anytime you want, first from where I should begin to start debugging and which files?

**Comment by lfuelling** [comment]
Hi, 
a fix for this has been released as `v3.3-rc.14`: https://github.com/gtasa-savegame-editor/gtasa-savegame-editor/releases/tag/v3.3-rc.14

Please test it if you find the time :)

**Comment by YousefSaber** [comment]
@lfuelling the fix works 👍.
